paws.common/R/client.R

```diff
@@ -64,10 +64,65 @@ Client <- struct(
 
 #-------------------------------------------------------------------------------
 
-# new_session returns a Session with default configuration.
-new_session <- function() {
+# Populate config
+merge_in_config <- function(dst, other){
+  if (is.null(other)){
+    return(dst)
+  }
+  # Populate credentials
+  if (!is.null(other$credentials)){
+    if (!is.null(other$credentials$creds$access_key_id)){
```

Comment:
I think if creds is provided, the existing one should be entirely overwritten.

Reply:
Addressed in my last commit